src/config.c

```diff
@@ -196,6 +196,24 @@ int loadConfig(char *configFile) {
             // Read next token...
             token = nextConfigToken();
             continue;
+        }
+        else if(strcmp("chroot", token)==0) {
+            // path is in next token
+            token = nextConfigToken();
+            snprintf(commonConfig.chroot, sizeof(commonConfig.chroot), "%s",
+              token);
```

Comment:
There is missing check for directory overflow / truncate. When directory path is too long it should not be silently truncated, instead some fatal error.

Reply:
Agreed! Return value of snprintf() is checked.